doc/nrf/known_issues.rst

```diff
@@ -161,6 +161,14 @@ NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
 Other issues
 ============
 
+.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-14312: During execution, in rare cases, the :c:func:`nrf_recv()` function would crash.
```

Comment:
nit: This is still present for these versions.
```suggestion
NCSDK-14312: During execution, in rare cases, the :c:func:`nrf_recv()` function crash.
```

Reply:
The wording is correct since "would" implies that it happens occasionally.